README.md

```diff
@@ -40,7 +40,7 @@ and ansible-doc to automate tasks on IMS.
 Ansible version compatibility
 ==============================
 
-This collection has been tested against the following Ansible versions: >=2.14.0,<2.16.0.
+This collection has been tested against the following Ansible versions: >=2.14.0.
```

Comment:
Instead of removing 2.16.0, consider changing it to 2.17 - so we can give them an idea for which range of versions have been tested.